README.md

```diff
@@ -44,6 +53,14 @@ $ cd spherely
 $ python -m pip install . -v --no-build-isolation
 ```
 
+Note that you can specify a build directory in order to avoid rebuilding the
+whole library from scratch each time after editing the code (requires
+scikit-build-core v0.2.0+):
+
+```
+$ python -m pip install . -v --no-build-isolation --config-settings build-dir=build/skbuild
```

Comment:
Yes, I started to use this yesterday and was planning to open a PR to add it to the readme ;)

You can use `skbuild/{cache_tag}` to have builds for multiple python versions next to each other, but maybe most contributors won't need that.